myFslInstallerScript.sh

```diff
@@ -0,0 +1,127 @@
+#!/bin/bash
+#myfslinstaller.sh
+
+set -e
+set -u
+
+#myfslinstaller function takes the parameters (OS and Version) and downloads the corresponding file of fsl installer from the server.
+if [[ $# -ne 0 && $# -ne 4 ]] ; then
+  echo -e "Please pass four valid parameters:(Version,OS,Installation Directory,Download Directory)\nUsage: myFslInstallerScript.sh 5.0.6 CentOS6 /usr/local/etc /usr/local/download/"
+  exit 1
+else 
+  echo "No parameters are passed. Taking default values."
```

Comment:
If I remove else, then there is no need to check for four parameters right ? It won't print anything since there are no parameters.